sdk/python/foundation-models/system/inference/image-classification/image-classification-batch-endpoint.ipynb

```diff
@@ -0,0 +1,504 @@
+{
+ "cells": [
```

Comment:
are there any gates for the notebooks/scripts to run them periodically?

Reply:
No, they are not set up as of now. 
I remember, @HrishikeshGeedMS was setting up workflow but was facing challenge in getting enough resources. Please @rjaincc check.

Reply:
@gauravrajguru , @raduk  - These workflows are the ones which are being ported to ADO.